website/content/docs/enterprise/sentinel.mdx

```diff
@@ -85,9 +103,43 @@ the Sentinel convention. Here are some examples:
 | `namespace.Description`                      | `namespace.description`                        |
 | `namespace.NodePoolConfiguration.Allowed[0]` | `namespace.node_pool_configuration.allowed[0]` |
 
+### Sentinel Node Objects
+
+The `node` object maps to the [Node][], with the fields converted to the
+Sentinel convention. Here are some examples:
+
+| Node Field         | Sentinel Accessor  |
+|--------------------|--------------------|
+| `node.Class`       | `node.class`       |
+| `node.Meta["foo"]` | `node.meta["foo"]` |
+
+### Sentinel Node Pool Objects
+
+The `node_pool` object maps to the [Node Pool][], with the fields converted to
+the Sentinel convention. Here are some examples:
+
+| Node Pool Field                                       | Sentinel Accessor                                       |
+|-------------------------------------------------------|---------------------------------------------------------|
+| `node_pool.Description`                               | `node_pool.description`                                 |
+| `node_pool.SchedulerConfiguration.SchedulerAlgorithm` | `node_pool.scheduler_configuration.scheduler_algorithm` |
+
+### Sentinel Dynamic Host Volume Objects
+
+The `volume` object maps to the [Dynamic Host Volume][], with the fields
+converted to the Sentinel convention. Here are some examples:
+
+| Dynamic Host Volume Field                    | Sentinel Accessor                              |
+|----------------------------------------------|------------------------------------------------|
+| `volume.Name`                                | `volume.name`                                  |
+| `volume.RequestedCapabilities[0].AccessMode` | `volume.requested_capabilities[0].access_mode` |
+
+
 [Nomad Sentinel Tutorial]: /nomad/tutorials/governance-and-policy/sentinel
 [`nomad sentinel`]: /nomad/docs/commands/sentinel
 [sentinel]: https://docs.hashicorp.com/sentinel
 [JSON job specification]: /nomad/api-docs/json-jobs
-[ACL token]: https://github.com/hashicorp/nomad/blob/v1.7.0-rc.1/nomad/structs/structs.go#L12991-L13020
-[Namespace]: https://github.com/hashicorp/nomad/blob/v1.7.0-rc.1/nomad/structs/structs.go#L5306-L5338
+[ACL token]: https://github.com/hashicorp/nomad/blob/v1.9.4/nomad/structs/structs.go#L13502-L13531
+[Namespace]: https://github.com/hashicorp/nomad/blob/v1.9.4/nomad/structs/structs.go#L5578-L5610
+[Node]: https://github.com/hashicorp/nomad/blob/v1.9.4/nomad/structs/structs.go#L2086-L2210
+[Node Pool]: https://github.com/hashicorp/nomad/blob/v1.9.4/nomad/structs/node_pool.go#L46-L68
+[Dynamic Host Volume]: https://github.com/hashicorp/nomad/blob/main/nomad/structs/host_volumes.go#L18-L87
```

Comment:
Note to reviewers: I hate that we have to link to code here rather than documentation but these objects are _not_ the `api` package objects. The `structs` version of these objects are subtly different.

Reply:
I don't suppose it's possible for the git ref to be dynamic based on which version is selected from the drop-down? also I presume we'll change this last one from `main` to `v.10.0` when it's released?

Reply:
Only by backporting the individual links. But yeah, this will be changed to 1.10.